sdk/spring/azure-spring-boot-test-aad-b2c/src/test/java/com/azure/test/aad/b2c/selenium/AADB2CSeleniumITHelper.java

```diff
@@ -36,98 +25,41 @@
         DEFAULT_PROPERTIES
             .put("azure.activedirectory.b2c.user-flows.profile-edit",
                 AADB2CTestUtils.AAD_B2C_PROFILE_EDIT);
-
-        final String directory = "src/test/resources/driver/";
-        final String chromedriverLinux = "chromedriver_linux64";
-        final String chromedriverWin32 = "chromedriver_win32.exe";
-        final String chromedriverMac = "chromedriver_mac64";
-        String osName = System.getProperty("os.name").toLowerCase();
-        Process process = null;
-        try {
-            File dir = new File(directory);
-            if (Pattern.matches("linux.*", osName)) {
-                process = Runtime.getRuntime().exec("chmod +x " + chromedriverLinux, null, dir);
-                process.waitFor();
-                System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, directory + chromedriverLinux);
-            } else if (Pattern.matches("windows.*", osName)) {
-                System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, directory + chromedriverWin32);
-            } else if (Pattern.matches("mac.*", osName)) {
-                process = Runtime.getRuntime().exec("chmod +x " + chromedriverMac, null, dir);
-                process.waitFor();
-                System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, directory + chromedriverMac);
-            } else {
-                throw new IllegalStateException("Unrecognized osName. osName = " + System.getProperty("os.name"));
-            }
-        } catch (InterruptedException | IOException e) {
-            throw new RuntimeException(e);
-        } finally {
-            if (process != null) {
-                process.destroy();
-            }
-        }
     }
 
-    public AADB2CSeleniumITHelper(Class<?> appClass, Map<String, String> properties) throws InterruptedException {
+    public AADB2CSeleniumITHelper(Class<?> appClass, Map<String, String> properties) {
         userEmail = AADB2CTestUtils.AAD_B2C_USER_EMAIL;
         userPassword = AADB2CTestUtils.AAD_B2C_USER_PASSWORD;
-        app = new AppRunner(appClass);
-        DEFAULT_PROPERTIES.forEach(app::property);
-        properties.forEach(app::property);
-        this.app.start();
-        setDriver();
-    }
-
-    public void quitDriver() {
-        try {
-            this.driver.quit();
-        } catch (Exception e) {
-            this.driver = null;
-        }
+        this.appClass = appClass;
```

Comment:
If there are no further references of appclass and properties in this class, we should not keep them as fields.